components/readme.adoc

```diff
@@ -743,6 +743,9 @@ Number of Components: 265 in 185 JAR artifacts (16 deprecated)
 | link:../camel-core/src/main/docs/timer-component.adoc[Timer] (camel-core) +
 `timer:timerName` | 1.0 | The timer component is used for generating message exchanges when a timer fires.
 
+| link:camel-twilio/src/main/docs/twilio-component.adoc[Twilio] (camel-twilio) +
+`twilio:apiName/methodName` | 2.20 | Represents a Twilio endpoint.
```

Comment:
Can you update the javadoc on the endpoint with a better description so the docs generated here have a better text